libs/client/features/src/accounts-manager/property/PropertyForm.tsx

```diff
@@ -36,7 +36,12 @@ export default function PropertyForm({ mode, defaultValues, onSubmit }: Props) {
             <section className="space-y-4 mb-8">
                 <h6 className="text-white uppercase">Location</h6>
                 <div className="space-y-4">
-                    <Input type="text" label="Country" value="United States" readOnly disabled />
+                    <Input
```

Comment:
This should be a dropdown list of all countries instead of a plain-text input.

Reply:
adjusted on latest commit